lec12-datasets.Rmd

```diff
@@ -0,0 +1,159 @@
+---
+title: "Introduction to the datasets, scientific method, and team dynamics; Start the projects"
+---
+
+-----
+
+## Lesson preamble:
+
+### Lesson objectives:
+
+- **Main goal**: 
+    - To start the process of identifying which dataset to use and some possible research questions
+- **Secondary goals**:
+    - Learn about the scientific method and *(the basics of)* how to apply it to your project
+    - Learn basics of group dynamics in a team setting, how to share responsibilities and duties
+    - Get into groups and start your projects
+
+### Lesson outline:
+
+Total lesson time: 2 hours
+
+- Datasets available for use (10-20 min)
+- What is the scientific method (20 min)
+- Team dynamics and assigning roles and tasks (15-20 min)
+- Break (10 min)
+- Getting into groups (10 min)
+    - Exercise (25-30 min)
+    - Start project work (20-25 min)
+
+-----
+
+## Datasets for use
+
+### List of datasets available
+
+{{list/link them here}}
+
+## Scientific method
+
+<!-- What is the scientific method? -->
+
+### Simplified diagram of the scientific method
+
+```{r fig_scientific_method, echo=FALSE}
+DiagrammeR::grViz('
+digraph rmarkdown {
+    graph [layout = neato, splines = true]
+    node [shape = box, style = rounded]
+    Hypothesis [pos = "0.75,3.75!"]
+    CollectData [label = "Collect Data", pos = "0.75,3!"]
+    AnalyzeData [label = "Analyze Data", pos = "0.75,2.25!"]
+    HypothesisTrue [label = "Hypothesis\nif TRUE", pos = "0,1.5!"]
+    HypothesisFalse [label = "Hypothesis\nif FALSE", pos = "1.5,1.5!"]
+    Report [label = "Report/Publish", pos = "0.75,0.75!"]
+
+    Hypothesis -> CollectData -> AnalyzeData -> HypothesisTrue -> Report
+    AnalyzeData -> HypothesisFalse -> {Report Hypothesis}
+}')
+```
+
+### Steps in the scientific process[^sci_method]
+
+1. Identify research question(s).
```

Comment:
I like first few bullets of the Khan academy guide you linked, because they emphasize that it starts with someone making an observation that generates a leads to a question, see something and then stop and think and eventually form an hypothesis.